presto-memory/src/test/java/io/prestosql/plugin/memory/TestMemoryMetadata.java

```diff
@@ -151,7 +151,7 @@ public void testReadTableBeforeCreationCompleted()
     public void testCreateSchema()
     {
         assertEquals(metadata.listSchemaNames(SESSION), ImmutableList.of("default"));
-        metadata.createSchema(SESSION, "test", ImmutableMap.of());
+        metadata.createSchema(SESSION, "test", ImmutableMap.of(), null);
```

Comment:
Can you pass something more meaningful here, `new Principal(USER, SESION.getUser())`?